src/tools/Makefile

```diff
@@ -22,9 +22,10 @@ PERL_MODULES += EPICS/PodXHtml.pm
 PERL_MODULES += Pod/Markdown.pm
 PERL_MODULES += URI/Escape.pm
 
-# This goes into lib/perl, not bin/<host>
+# This also goes into lib/perl
 PERL_MODULES += EpicsHostArch.pl
 
+PERL_SCRIPTS += EpicsHostArch.pl
```

Comment:
Adding `EpicsHostArch.pl` to `PERL_SCRIPTS` installs it into the `bin/$(EPICS_HOST_ARCH)` directory. There's a bootstrapping problem there, if you know which `bin` subdirectory it's in, why do you need to run the script at all?